helper/sshutil/sshutil.go

```diff
@@ -129,6 +143,21 @@ func (sw *SSHSessionWrapper) RunCommand(ctx context.Context, cmd string) error {
 // - either a path to the private key file,
 // - or the content or this private key file
 func ReadPrivateKey(pk string) (ssh.AuthMethod, error) {
+	raw, err := ToPrivateKeyContent(pk)
+
+	signer, err := ssh.ParsePrivateKey(raw)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Failed to parse key file %q", pk)
+	}
+	if err != nil {
```

Comment:
Looks like the missing error check above

Reply:
Oups